geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSet.java

```diff
@@ -357,6 +357,27 @@ long zcount(SortedSetScoreRangeOptions rangeOptions) {
     return addLimitToRange(rangeOptions, withScores, minIndex, maxIndex);
   }
 
+  long zlexcount(SortedSetLexRangeOptions lexOptions) {
+    // Assume that all members have the same score. Behaviour is unspecified otherwise.
+    double score = scoreSet.get(0).score;
```

Comment:
Before you call get(0) do you need to do a check on scoreSet to see if it is empty?

Reply:
Done